tests/test_syntax/inline/test_advanced_links.py

```diff
@@ -0,0 +1,98 @@
+from markdown.test_tools import TestCase
+
+
+class TestAdvancedLinks(TestCase):
```

Comment:
We suppose we can leave the class named "advanced" but let's just name the file `links.py`. Eventually all "kink" related tests should be moved here.

Reply:
Sounds good.